source/docs/software/python/deploying-robot-code.rst

```diff
@@ -0,0 +1,75 @@
+
+Deploying Robot Code
+====================
+
+.. note:: Before deploying the code to your robot, you must start by :ref:`installing RobotPy on your roboRIO <docs/software/python/installing-robotpy-on-roborio:RobotPy installation on roboRIO>`
+
+To deploy code to your robot, you can just run the following command and it will upload the code and start it immediately.
+
+.. tabs::
+   .. group-tab:: Windows
+
+    .. code-block:: sh
+
+        py -3 robot.py deploy
+
+   .. group-tab:: Linux/macOS
+
+    .. code-block:: sh
+
+        python3 robot.py deploy
+
+You can watch your robot code's output (and see any problems) by using the netconsole program (you can either use NI's tool, or `pynetconsole <https://github.com/robotpy/pynetconsole>`__. You can use netconsole and the normal FRC tools to interact with the running robot code.
+
+If you're having problems deploying code to the robot, you can `join the Discord channel <https://discord.gg/Vj3DWnBr>`__ and ask for help!
+
+Immediate feedback via Netconsole
+---------------------------------
+
+Note that when you run the deploy command like that, you won't get any feedback from the robot whether your code actually worked or not. If you want to see the feedback from your robot without launching a separate NetConsole window, a really useful option is :guilabel:`--nc`. This will cause the deploy command to show your program's console output, by launching a netconsole listener.
+
+.. tabs::
+
+   .. group-tab:: Windows
+
+    .. code-block:: sh
+
+        py -3 robot.py deploy --nc
+
+   .. group-tab:: Linux/macOS
+
+    .. code-block:: sh
+
+        python3 robot.py deploy --nc
```

Comment:
The indentation here is a little weird. In the docs, we standardize on 3 space indents.